llama_cpp/llama.py

```diff
@@ -814,60 +814,32 @@ def _create_completion(
                             break
 
                 token_end_position = 0
-                for token in remaining_tokens:
-                    token_end_position += len(self.detokenize([token]))
```

Comment:
some CJK character contains several tokens,  while it only decodes a single token, which causes this bug